src/apiutils.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
Can we rename the file to oasutils.js??
As most of the methods are scoped to the complete oasDoc rather than just the api endpoints.

Reply:
I don't have a preference. Just give a meaningful name that you are happy with.

Reply:
Addressed in [https://github.com/googleinterns/OAS-Adherence-Test-Suite/pull/4](url)